lib/gooddata.rb

```diff
@@ -8,8 +8,8 @@
 $pmap_default_thread_count = 20 # rubocop:disable GlobalVars
 
 # GoodData Module
-module GoodData
-end
+ module GoodData
+end 
```

Comment:
#<struct Pronto::Git::Line line=#<Rugged::Diff::Line:47139890207460 {line_origin: :addition, content: "end \n">, patch=#<struct Pronto::Git::Patch patch=#<Rugged::Patch:47139881659720>, repo=#<Pronto::Git::Repository:0x000055bf37cd5dd8 @repo=#<Rugged::Repository:47139881660120 {path: "/src/.git/"}>>>, hunk=#<Rugged::Diff::Hunk:47139890208400 {header: "@@ -8,8 +8,8 @@ require 'pmap'\n", count: 10}>> warning: Lint/EndAlignment: `end` at 12, 0 is not aligned with `module` at 11, 1. (rubocop)